.github/workflows/coverage.yml

```diff
@@ -0,0 +1,40 @@
+name: Code Coverage
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+
+    steps:
+
+    - name: Set up Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.15
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Basic build
+      run: go build ./cmd/...
+
+    - name: Run tests on linux with coverage data
+      run: go test -race -covermode=atomic -coverprofile=coverage.txt -bench=regex ./...
```

Comment:
I think you want `bench=.` and `-benchtime=1x`.

Do we need coverage counts? If not, we can omit covermode, I believe.

Reply:
Done.

Now I understand what -bench=regex means, it is supposed to be the regex not putting it into a regular expression mode of some kind.